### ISSUE TITLE
Is there a way to run multiple guppy_basecaller jobs in parallel?

### ISSUE THREAD

**Issue by benbfly** [nf-core/nanoseq#163]
## Is your feature request related to a problem? Please describe

We have a multi-GPU server where each process can only access a single GPU.  So it is very advantageous for us to run multiple guppy_basecaller instances in parallel (usually we batch 50 or so fast5 files in a single process) and merge the results. From what I can surmise from the sample sheet description, this is not a workflow that is currently supported by nanoseq. Is that correct?

## Describe the solution you'd like

Is there a pattern for that from other nf-core workflows which use embarrassingly parallel processing of reads?

## Describe alternatives you've considered

I guess we could hack together a solution by creating many nanoseq runs for the basecalling, and then merging those together using nanoseq with basecalling skipped.  But it would be nice to have this built in.

Thanks!

**Comment by christopher-hakkaart** [comment]
Guppy was removed in 3.1 release #222
